nexus/reconfigurator/planning/src/mgs_updates/test_helpers.rs

```diff
@@ -107,7 +110,7 @@ pub(super) const ARTIFACT_HASH_ROT_BOOTLOADER_PSC: ArtifactHash =
 pub(super) const ARTIFACT_HASH_ROT_BOOTLOADER_SWITCH: ArtifactHash =
     ArtifactHash([28; 32]);
 /// Hash of fake artifact for host OS phase 1
-pub(super) const ARTIFACT_HASH_HOST_PHASE_1: ArtifactHash =
+pub(super) const ARTIFACT_HASH_GIMLET_HOST_PHASE_1: ArtifactHash =
     ArtifactHash([29; 32]);
 /// Hash of fake artifact for host OS phase 1 (for a fake version 1.5)
 pub(super) const ARTIFACT_HASH_HOST_PHASE_1_V1_5: ArtifactHash =
```

Comment:
Nit: For consistency, it'd be nice to have this and any other hash that is gimlet-specific, say it in the name like the other constants that were changed.